lib/util/stream_utils.js

```diff
@@ -812,7 +828,7 @@ shaka.util.StreamUtils = class {
       const video = variant.video;
       if (audio && currentVariant && currentVariant.audio) {
         if (!StreamUtils.areStreamsCompatible_(audio, currentVariant.audio)) {
-          shaka.log.debug('Droping variant - not compatible with active audio',
+          shaka.log.debug('Dropping variant - not compatible with active audio',
```

Comment:
Can you add a rule at https://github.com/shaka-project/shaka-player/blob/main/build/misspellings.txt to prevent it from happening in the future? Thanks!